tests/integration-tests/tests/test_efa/test_efa/test_efa/pcluster.config.ini

```diff
@@ -0,0 +1,22 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = t2.micro
```

Comment:
I would go for a bigger instance for the master cause t2.micros are very slow and might take quite a lot to be initialised.

Reply:
changed to `c5.xlarge`